setup.cfg

```diff
@@ -83,8 +83,8 @@ tests =
   pytest-doctestplus>=0.5 # We require the newest version of doctest plus to use +IGNORE_WARNINGS
   pytest-mock
   pytest-mpl>=0.12 # First version to support our figure tests
-  pytest-xdist>=1.27
-  pytest>=5.4
+  pytest-xdist>=2.0
+  pytest>=6.0
```

Comment:
Didn't we have a conversation recently about pytest versions because of debain or something?

Ah it was this: https://github.com/sunpy/sunpy/issues/5904 looks fine then.